### ISSUE TITLE
Generating Lit 3 API requires a manual change to the `ChildPart` export to work.

### ISSUE THREAD

**Issue by AndrewJakubowicz** [lit/lit.dev#1112]
### Issue

Currently TypeDoc is unable to find the exported type for `ChildPart`. 

### Repro

Try and run the `lit-3` config without any changes to the built Lit v3 repo. E.g. `npm run build --workspace lit-dev-api`.
Note the following error: `Cannot find ChildPart`.

### Workaround

In `lit-html.d.ts` within `lit.dev/packages/lit-dev-api/api-data/lit-3/repo/packages/lit-html/development/lit-html.d.ts`,

change:

```ts
export type { ChildPart };
declare class ChildPart implements Disconnectable {
```

to

```ts

export declare class ChildPart implements Disconnectable {
```

### Fix

This issue is closed once we can automatically build the Lit 3.0 API docs without any manual intervention. Due to workaround this isn't blocking.

**Comment by augustjk** [comment]
i wonder if somehow typedoc doesn't do the hoisting. perhaps we can move the `export type {ChildPart}; to be below the class declaration?

**Comment by augustjk** [comment]
Fixed by #1119
